### DISCUSSION TITLE
Mobile port?

### DISCUSSION THREAD

**Discussion by YARusK** [MinicraftPlus/minicraft-plus-revived#699] [Q&A]
This game will be perfect on mobile phones

**Comment by BenCheung0422** [comment]
For some information, you may head over [my recent blog](https://minicraft.wiki.gg/wiki/User_blog:Benforge0422/How_We_Should_Do_Mobile_Port_of_Minicraft%2B).

**Comment by ErickN13** [comment]
radiofrequency/diaf

I knew this port for Android that it carried but it wasn't MinicraftPlus, but you can use it as a base.
